extension/src/popup/components/manageAssets/SelectAssetRows/index.tsx

```diff
@@ -45,6 +45,17 @@ export const SelectAssetRows = ({ assetRows }: SelectAssetRowsProps) => {
     return "";
   };
 
+  const getTokenBalanceFromCanonical = (canonical: string) => {
```

Comment:
small unrelated fix: when choosing an asset to Send Payment with, token balance wasn't being formatted with decimals correctly